kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/KafkaServiceReconcilerIT.java

```diff
@@ -234,7 +234,7 @@ private void assertResolvedRefsTrue(KafkaService cr, String expectedBootstrap) {
     private void assertResolvedRefsFalse(KafkaService cr,
                                          String reason,
                                          String message) {
-        AWAIT.alias("FilterStatusResolvedRefs").untilAsserted(() -> {
+        AWAIT.alias("KafkaServiceStatusResolvedRefs").untilAsserted(() -> {
```

Comment:
I wonder what value we're getting from the alias. IIRC you don't need to call it. It might be better to remove all these calls than have to keep these things correct everywhere.

Reply:
I don't disagee with you.  I'd be happy to kill it (in another PR).